docs/source/explanation/Main_algorithms/index.rst

```diff
@@ -0,0 +1,12 @@
+Main algorithms
+===============
+
+.. toctree::
+   :maxdepth: 1
+
```

Comment:
I'm trying to figure out how to reveal the classification subsections, but not headings of anomaly classification and so on. If we have maxdepth:1, we can't see the classification sections, what can be confusing (1st image)
![image](https://user-images.githubusercontent.com/33455576/210800977-69cf114f-f990-4183-85fe-4be3a58f6faa.png). 

If we have maxdepth:2, it's also confusing (2nd image)
![image](https://user-images.githubusercontent.com/33455576/210801250-ef61a310-7309-4d9f-ad02-de8bfca034ea.png). 

Do you know, how we could combine both approaches here?

Reply:
If no, let's keep it as it is

Reply:
You can try 
.. toctree::
   :titlesonly:
   :maxdepth:2

Reply:
Thank you, I will try!

Reply:
It works perfectly, thanks